test/data/anki-note-builder-test-results.json

```diff
@@ -3768,7 +3805,7 @@
         "cloze-body-kana": "すっっごーーい",
         "cloze-prefix": "cloze-prefix",
         "cloze-suffix": "cloze-suffix",
-        "conjugation": "",
```

Comment:
why is this marked as ksb 🤔
I'll have to figure this out.

Reply:
This happens on master too when katakanaToHiragana and collapseEmphatic are on (see discord), so I considered it out of the scope of this.